helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixAdmin.java

```diff
@@ -1315,8 +1315,11 @@ public void addTypeToCustomizedStateConfig(String clusterName, String type) {
     ZKHelixDataAccessor accessor =
         new ZKHelixDataAccessor(clusterName, new ZkBaseDataAccessor<ZNRecord>(_zkClient));
     PropertyKey.Builder keyBuilder = accessor.keyBuilder();
-    accessor.updateProperty(keyBuilder.customizedStateConfig(),
-        customizedStateConfigFromBuilder);
+    if(!accessor.updateProperty(keyBuilder.customizedStateConfig(),
```

Comment:
This pull only deals with exception in group commit. How about the zkclient closed issue in callbackhandler?

Reply:
The reason we need this PR is that we need to make sure updateProperty with group commit or non group commit have the same behavior. Currently non group commit update return error code, while group commit throws exception. This should not impact any callbackhandler existing logic.